.travis.yml

```diff
@@ -10,11 +10,29 @@ os:
 
 install:
 - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install make; fi
-- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get install make; fi
+- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+    sudo apt-get install make;
+    sudo apt-get update -qq;
+    sudo apt-get install -qq -y software-properties-common uidmap;
+    sudo add-apt-repository -y ppa:projectatomic/ppa;
+    sudo apt-get update -qq;
+    sudo apt-get -qq -y install podman;
+  fi
 
-script:
-- if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then mingw32-make build; fi
-- if [[ "$TRAVIS_OS_NAME" != "windows" ]]; then make unit build; fi
+script: |
+  if [[ "$TRAVIS_OS_NAME" == "windows" ]]; then
+    mingw32-make build
+  fi
+  if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then
+    make unit build
+  fi
+  if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+    make unit build
+    go build -mod=vendor -o bin/opm ./cmd/opm
+    go build -mod=vendor -o bin/initializer ./cmd/initializer
+    go run opm-test.go docker
```

Comment:
Any reason to use `go run` instead of `go test` here? I think at the very least with `go test` we get some nice configuration and output stuff for free.